source/includes/_transactions.md

```diff
@@ -53,13 +58,12 @@ message Payload {
 {
     "commands": [
         {
-            "command_type": string(?),
-            /* other command-specific fields */
+            /* command-specific fields */
```

Comment:
The example for the valid transaction is provided above, this is used for explanation.  Is it necessary to complicate it, as it might hurt the explanation?

Reply:
Concern is that person can just try to copy and paste the example and try to run it with iroha-cli. Maybe this explanation can be done in text, and not in JSON?